pkg/apis/core/v1beta1/types_seed.go

```diff
@@ -134,12 +134,9 @@ type Backup struct {
 	// Region is a region name. This field is immutable.
 	// +optional
 	Region *string `json:"region,omitempty" protobuf:"bytes,3,opt,name=region"`
-	// SecretRef is a reference to a Secret object containing the cloud provider credentials for
-	// the object store where backups should be stored. It should have enough privileges to manipulate
-	// the objects as well as buckets.
-	// Deprecated: This field will be removed after v1.121.0 has been released. Use `CredentialsRef` instead.
-	// Until removed, this field is synced with the `CredentialsRef` field when it refers to a secret.
-	SecretRef corev1.SecretReference `json:"secretRef" protobuf:"bytes,4,opt,name=secretRef"`
+
+	// SecretRef is tombstoned to show why 4 is reserved protobuf tag.
+	// SecretRef corev1.SecretReference `json:"secretRef" protobuf:"bytes,4,opt,name=secretRef"`
```

Comment:
Usually, we add a test to ensure this tag doesn't get reintroduced: https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/types_shoot_test.go

Reply:
Addressed with 4b4150be17